Week1/homework/queries.js

```diff
@@ -0,0 +1,114 @@
+var mysql      = require('mysql');
+var connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'myworld'
+});
+
+connection.connect(function(err) {
+    if (err) throw err;
+    console.log('db connected');
+  });
+
+// Question 4i: What are the names of the countries with population greater than 8 million
+let result4i = 'SELECT Country_Name FROM Country WHERE Population > 8000000;';
+
+connection.query(result4i, function(error, data) {
+  if (error) {
+    throw error;
+  }
+  console.log(data);
+});
+
+// Question 4ii: What are the names of the countries that have “land” in their names ?
+let result4ii = "SELECT Country_Name FROM Country WHERE Country_Name like '%land%';";
+
+connection.query(result4ii, function(error, data) {
+  if (error) {
+    throw error;
+  }
+  console.log(data);
+});
+
+// Question 4iii: What are the names of the cities with population in between 500,000 and 1 million ?
+let result4iii = "SELECT City_Name FROM City WHERE Population BETWEEN 500.000 AND 1000000;";
```

Comment:
Careful 500.000 will be considered as float so basically 500 and not as Int 500000.